webapp/src/components/common/EditableMatrix/style.ts

```diff
@@ -6,7 +6,7 @@ export const Root = styled(Box)(({ theme }) => ({
   display: "flex",
   flexDirection: "column",
   alignItems: "center",
-  overflow: "auto",
+  overflow: "hidden",
```

Comment:
Are you sure you don't break anything? Test with big matrix.

Reply:
Handsontable docs recommands the parent container overflow is set to "hidden", normally it doesn't break anything i've tested

Reply:
Doc recommands `auto` or `hidden`. If you're sure ok.